spec/poll_spec.rb

```diff
@@ -26,6 +26,14 @@
         expect { poll.add_vote(vote) }.to raise_error Poll::InvalidCandidateError
       end
     end
+
+    context 'reject when vote time passed the deadline' do
```

Comment:
> テストの内容を表す英語が適切か見てほしい

context に書くのは前提条件のちがいです。
`reject` を入れてしまうと、振る舞いについても書いてしまっていることになるのでこれは不要かなと思います。

```suggestion
    context 'when vote time passed the deadline' do
```